### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by folmos-at-orange**
- Change "khiops-python" to "Khiops Python library"
- Eliminate old conda-build documentation
- Other minor changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -71,10 +71,10 @@
 
 ## 10.2.0.0 - 2024-02-15
 Note: This release marks the open sourcing of Khiops:
-- The `khiops-python` replaces the old `pykhiops` package. We recommend to uninstall `pykhiops`
-  before installing `khiops-python`. More information at the [Khiops website][khiops].
+- The `khiops-python` Python package replaces the old `pykhiops` package. We recommend to uninstall
```
Comment (popescu-v) [inline CHANGELOG.md:74]: We don't install `khiops-python`, but Khiops (the Python library is called `khiops`). The packages (Conda, Pip) are also called `khiops`. Only the Git repository is called `khiops-python`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -71,10 +71,10 @@
 
 ## 10.2.0.0 - 2024-02-15
 Note: This release marks the open sourcing of Khiops:
-- The `khiops-python` replaces the old `pykhiops` package. We recommend to uninstall `pykhiops`
-  before installing `khiops-python`. More information at the [Khiops website][khiops].
+- The `khiops` package replaces the old `pykhiops` package. We recommend to uninstall
+  `pykhiops` before installing `khiops-python`. More information at the [Khiops site][khiops].
```
Comment (popescu-v) [inline CHANGELOG.md:75]: s/`khiops-python`/the Khiops Python library/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -28,11 +28,11 @@ python -m unittest tests.test_sklearn.KhiopsSklearnParameterPassingTests
 ## Coding Style
 
 ### Language
-`khiops-python` is coded in English (comments included).
+We code in English (comments included).
 
 ### Style Enforcement: Tools & Git Hooks
-The coding style of pyKhiops is almost PEP8 compliant. To enforce these guidelines we use three
-tools:
+The coding style of the khiops library is almost PEP8 compliant. To enforce these guidelines we use
```
Comment (popescu-v) [inline CONTRIBUTING.md:34]: s/Khiops library/Khiops Python library/

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -243,8 +243,8 @@ carefree while still trying to not add too many dependencies.
 ## Versioning
 We follow a non-standard `MAJOR.MINOR.PATCH.INCREMENT[PRE_RELEASE]` versioning convention. The
 first three numbers `MAJOR.MINOR.PATCH` are the latest Khiops version that is compatible with the
-package. The number `INCREMENT` indicates the evolution of `khiops-python` followed by an optional
-`[PRE_RELEASE]` version for alpha, beta and release candidate releases (eg. `b2`).
+package. The number `INCREMENT` indicates the evolution of the Khiops Python library followed by an
+optional `[PRE_RELEASE]` version for alpha, beta and release candidatejreleases (eg. `b2`).
```
Comment (popescu-v) [inline CONTRIBUTING.md:247]: s/`b2`/`b.2`/ (we now support dot between the letter and the number in the pre-release version field).

File: README.md [hunk 5]
```diff
@@ -31,7 +31,8 @@ Other install method are documented at the [Khiops website][khiops-install].
 [sklearn]: https://scikit-learn.org/stable
 
 ## Documentation
-The main documentation of the Khiops Python library is available [here][khiops-python-doc].
+The API docs for the Khiops Python library is available [here][khiops-api-docs]. Other documentation
```
Comment (popescu-v) [inline README.md:34]: s/the API docs ... is available [here]/the API documentation ... is available [here]/
OR
s/the API docs ... is available [here]/the API documentation ... are available [here]/

File: khiops/tools.py [hunk 6]
```diff
@@ -78,7 +78,7 @@ def kh_samples_entry_point():  # pragma: no cover
 
         argument_parser = samples_module.build_argument_parser(
             prog="kh-samples [core|sklearn]",
-            description="Executes the sample code snippets of khiops-python",
+            description="Executes the sample code snippets of the Khiops library",
```
Comment (popescu-v) [inline khiops/tools.py:81]: s/Khiops library/Khiops Python library/

**Review by popescu-v — CHANGES_REQUESTED** [review]
See comments.

File: CHANGELOG.md (hunk 2 above)
Comment (folmos-at-orange) [reply CHANGELOG.md:75]: Ok & Done.

File: CONTRIBUTING.md (hunk 3 above)
Comment (folmos-at-orange) [reply CONTRIBUTING.md:34]: Ok & Done

File: CONTRIBUTING.md (hunk 4 above)
Comment (folmos-at-orange) [reply CONTRIBUTING.md:247]: `b.2` doesn't work with conda.

File: README.md (hunk 5 above)
Comment (folmos-at-orange) [reply README.md:34]: API Docs is widely used so this is fine.

File: khiops/tools.py (hunk 6 above)
Comment (folmos-at-orange) [reply khiops/tools.py]: OK & Done

File: CONTRIBUTING.md (hunk 4 above)
Comment (popescu-v) [reply CONTRIBUTING.md]: Oh yes it does! See e.g. https://anaconda.org/khiops-dev/khiops-core/10.2.3b.1/download/win-64/khiops-core-10.2.3b.1-hc1dd92d_0.tar.bz2. What doesn't work with Conda is the dash, e.g. "10.2.3-b.1" needs to be converted to "10.2.3b.1" for Conda package versioning.

File: README.md (hunk 5 above)
Comment (popescu-v) [reply README.md]: Ok, then "API docs .. _are_ available"

File: packaging/conda/README.md [hunk 7]
```diff
@@ -15,22 +13,6 @@ KHIOPS_REVISION=10.2.0
 conda build --output-folder ./khiops-conda-build packaging/conda
 
 # Linux/macOS
-# Note: We need the conda-forge channel to obtain the pinned versions of MPICH
+# Note: We use the conda-forge channel so the khiops-core package obtains the pinned MPICH versions
 conda build --channel conda-forge --output-folder ./khiops-conda-build packaging/conda
 ```
-
-### Executable Signatures in macOS
```
[the remark this answers is not in the document]
Comment (folmos-at-orange) [reply packaging/conda/README.md:22]: Ok & Done.

File: khiops/tools.py [hunk 8]
```diff
@@ -78,7 +78,9 @@ def kh_samples_entry_point():  # pragma: no cover
 
         argument_parser = samples_module.build_argument_parser(
             prog="kh-samples [core|sklearn]",
-            description="Executes the sample code snippets of khiops-python",
+            description=(
+                "Executes the sample code snippets of " "the Khiops Python library"
```
[the remark this answers is not in the document]
Comment (folmos-at-orange) [reply khiops/tools.py:82]: Ok & Done.

File: CONTRIBUTING.md (hunk 4 above)
Comment (folmos-at-orange) [reply CONTRIBUTING.md]: Ok & Done.

**Pull request merged**